frontend/src/components/app/LoginForm.js

```diff
@@ -200,7 +199,7 @@ class LoginForm extends Component {
         if (roleSelect) {
           return loginCompletionRequest(role)
             .then(() => {
-              dispatch(loginSuccess(auth));
+              auth.login();
```

Comment:
Not sure I follow exactly why we are ditching the actions dispatching and not doing the logic in the redux actions where this should reside.

Reply:
How will we "know" we are authenticated at any point in time throughout the app at any level ? Do we need to pass the auth anywhere ? Can't we use just a flag from the redux store ?

Reply:
I don't think we've ever used this flag tbh. We were using the value from the local storage. So it's a duplication of concerns. If you'll check how `auth.login` works, it does pretty much the same but in a clean way. All the requests and actions, that were called before are still fired.